.asf.yaml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+notifications:
+  commits:      commits@kafka.apache.org
+  issues:       github@kafka.apache.org
+  pullrequests: github@kafka.apache.org
```

Comment:
Do we have this mailing list already?

Reply:
Good Question. I think, we dont have ``github@kafka.apache.org` yet.

Only ASF members and PMC Chair can create mailing list using : https://selfserve.apache.org/

Maybe we can request @junrao to create a new mailing address : `github@kafka.apache.org`

Reply:
Can we use the `commits` mailing list for now?

Reply:
Btw, we have a `JIRA mailing list`, maybe we can use that for `issues`.

Reply:
sounds good. we can use `jira@afka.apache.org`. If required, we can change later.
updated the PR.